tests/code/test_events.py

```diff
@@ -479,9 +479,25 @@ def test_deleteEvent():
         for count, recurring in enumerate(recurringEventsAfter):
             assert recurring.name == ("Not Empty Bowls Spring Week " + str(count + 1))
         assert (len(recurringEventsBefore)-1) == len(recurringEventsAfter)
-
         transaction.rollback()
 
+        #creating recurring event again to test def deleteRecurringSeries()
+        eventInfo['valid'] = True
+        eventInfo['program'] = Program.get_by_id(1)
+        recurringEvents = saveEventToDb(eventInfo)
+        event = Event.get_by_id(recurringEvents[1].id)
+        recurringId = event.recurringId
+
+        totalRecurringEvents = len(Event.select().where(Event.recurringId == recurringId))
+        #checks the number of all recurring events that will take place after a recurring event plus the event itself.
+        eventPlusAllRecurringEventsAfter = len(Event.select().where((Event.recurringId == recurringId) & (Event.startDate >= event.startDate)))
+
+        with app.app_context():
+            g.current_user = User.get_by_id("ramsayb2")
+            deletingEvent = deleteEventAndAllFollowing(recurringEvents[1])
```

Comment:
I think that if you are wanting to test the delete current and all following function, it would be better to not delete the first event in the series since that is basically just checking that you are deleting the whole series (which is now its own separate function). I think that it would be better to delete an event in the middle of the series and make sure that the older ones are still there.

Reply:
fixed!